RELEASE/scripts/autoscend/auto_restore.ash

```diff
@@ -1628,7 +1645,7 @@ void invalidateRestoreOptionCache()
  */
 boolean acquireMP()
 {
-	return acquireMP(my_maxmp());
+	return acquireMP(min(0.95 * my_maxmp(),300));
```

Comment:
acquireMP() without a parameter should automatically come up with areasonable amount of MP instead of restoring to max possible.
I think the minimum between 95% and 300 sounds reasonable enough. maybe could go to 80% or 300.

note that this will not break anything. currently acquireMP() with no parameters is only used once in the code. where i put it incorrectly. and fixed it in #612 where I switch it for `acquireMP(200)`